src/components/Issues.js

```diff
@@ -1,9 +1,12 @@
 import React, {useState, useEffect} from "react";
-import {FlexCenter} from "../styles/Grid";
+import {Flex, FlexCenter} from "../styles/Grid";
 import {EmptyPlaceholder} from "../styles/EmptyPlaceholder";
 import api from "../lib/apiGraphQL";
 import Card from "./Card";
 import List from "../styles/List";
+import Button from "../styles/Button";
+import Paragraph from "../styles/Paragraph";
+//import Container from "../styles/Container";
```

Comment:
missed this one 😛

Reply:
Good catch! my struggles were evident, lol